tox.ini

```diff
@@ -90,3 +90,4 @@ relative_files = True
 [coverage:report]
 exclude_lines =
     \.\.\.
+    pragma: no cover
```

Comment:
how this works? we still can see that those lines are not covered, right?

Reply:
We can see that they're not covered, but they won't be included in reports if we mark a scope with `# pragma: no cover`.